CHANGELOG.md

```diff
@@ -16,6 +16,12 @@
 
 ---
 
+## v2.2.1
+
+- ✨ Implement addTiming ([#668](https://github.com/DataDog/browser-sdk/pull/668))
+- [logs] document beforeSend API ([#664](https://github.com/DataDog/browser-sdk/pull/664))
+- v2.2.0 ([#678](https://github.com/DataDog/browser-sdk/pull/678))
```

Comment:
we generally only keep entries that could be interesting for customers, so no need to list doc changes or previous version PR